panda/src/eagldisplay/PandaViewController.h

```diff
@@ -0,0 +1,14 @@
+#import <UIKit/UIKit.h>
```

Comment:
Let's put file headers in the Objective-C files as well, and also stick to the regular naming conventions (`pandaViewController.h`, `eaglGraphicsWindow.h`).